docs/dev/api_integration.md

```diff
@@ -1,170 +1,70 @@
-| Type               | MP4 H.264                                                                             |
-| Size               | The maximum video size is 1GB.                                                        |
-| Dimensions         | The aspect ratio must be 16:9 exactly and the resolution must be 1920x1080 or higher. |
-| Name               | The video file name must be `partnerName-appName.mp4`.                                |
-| Video Length       | The maximum video length is 60 seconds.                                               |
-| Description        | The maximum number of characters allowed is 300.                                      |
-
-Technology Partners can add up to eight images (seven if you are including a video) in an integration tile's media carousel.
 
-The images must meet the following requirements:
+You can also include out-of-the-box assets such as [monitors][25], [dashboards][26], and [log pipelines][27] with your integration. When a user clicks **Install** on your integration tile, they are prompted to follow the setup instructions, and all out-of-the-box dashboards and recommended monitors packaged with your integration will appear in the their account.
 
-| Image Requirements | Description                                                                                                                                       |
-|--------------------|---------------------------------------------------------------------------------------------------------------------------------------------------|
-| Type               | `.jpg` or `.png`.                                                                                                                                 |
-| Size               | The average is around 500KB. The maximum image size is 1MB.                                                                                       |
-| Dimensions         | The aspect ratio must be 16:9 exactly and fit these specifications:<br><br>- Width: 1440px<br>- Minimum height: 810px<br>- Maximum height: 2560px |
-| Name               | Use letters, numbers, underscores, and hyphens. Do not use spaces.                                                                           |
-| Color Mode         | RGB                                                                                                                                               |
-| Color Profile      | sRGB                                                                                                                                              |
-| Description        | The maximum number of characters allowed is 300.                                                                                                  |
+Since API integrations do not use the Datadog Agent to collect data, you need to create a tile (pictured below) to display your offering on the **Integrations** page once your development work is complete.
```

Comment:
This line isn't accurate. All partner integrations require a tile, not just API integrations. 

I think the previous copy was clearer. For API integrations, the partner needs to add the steps to configure the integration in their tile.